src/xdocs/config_system_properties.xml

```diff
@@ -128,6 +119,10 @@ Audit done.
 Checkstyle ends with 2 errors.
           </source>
         </p>
+        <p>
+          Attention to Windows users: system properties should be wrapped with <code>"</code>.
+          Example: <code>-D"&lt;property&gt;=&lt;value&gt;"</code>
+        </p>
```

Comment:
I think it is better to have this in "Notes" section

Reply:
We should make "General" section.
Briefly describe how to do this for CLI.
And have Note subsection for windows users.